src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1873,16 +1887,19 @@ private void printDevModeMessages(boolean inputUnavailable, boolean startup) {
             hotTests = true;
         }
         if (startup) {
+            info(formatAttentionTitle("Liberty server port information:"));
```

Comment:
I think it would be good to add an empty line before this section, so that the distinction between the dev mode actions and server ports sections are clearer.   i.e. `info(formatAttentionMessage(""))`